python/eos/tasks.py

```diff
@@ -484,12 +484,12 @@ def sample_nested(analysis_file:str, posterior:str, base_directory:str='./', bou
     :type seed: int, optional
     """
     analysis = analysis_file.analysis(posterior)
-    results = analysis.sample_nested(bound=bound, nlive=nlive, dlogz=dlogz, maxiter=maxiter, seed=seed)
-    samples = results.samples
-    posterior_values = results.logwt - results.logz[-1]
-    weights = _np.exp(posterior_values)
-    eos.data.DynestyResults.create(os.path.join(base_directory, posterior, 'dynesty_results'), analysis.varied_parameters, results)
-    eos.data.ImportanceSamples.create(os.path.join(base_directory, posterior, 'samples'), analysis.varied_parameters,
+    for iter, results in enumerate(analysis.sample_nested(bound=bound, nlive=nlive, dlogz=dlogz, maxiter=maxiter, seed=seed)):
+        samples = results.samples
+        posterior_values = results.logwt - results.logz[-1]
+        weights = _np.exp(posterior_values)
+        eos.data.DynestyResults.create(os.path.join(base_directory, posterior, f'dynesty_results-{iter:04}'), analysis.varied_parameters, results)
+        eos.data.ImportanceSamples.create(os.path.join(base_directory, posterior, f'samples-{iter:04}'), analysis.varied_parameters,
```

Comment:
I like the labelling of the dynesty result by the iteration number. However, doing so for the samples will break backward compatibility with e.g. the plotting framework. Please undo this labelling for ``samples``.